py_gtfs_rt_ingestion/pyproject.toml

```diff
@@ -12,6 +12,7 @@ boto3 = "^1.23.3"
 
 [tool.poetry.dev-dependencies]
 pytest = "^5.2"
+pandas = "^1.4.2"
```

Comment:
Add pandas as dev dependency in poetry, I think pyarrow already includes pandas, but better to be explicit.